### PR TITLE
Imprv/bitwise flags

### PR DESCRIPTION
**Pull request by yaaminu**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,45 +1,54 @@
 const FLAGS = {
```
Comment (vigan-abd) [inline index.js:3]: please add use strict as well at beginning

**Review by vigan-abd — CHANGES_REQUESTED** [review]
function lgtm, minor request

**Review by vigan-abd — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -6,7 +6,21 @@
...
   ],
-  "dependencies": {
+  "scripts": {
+    "test": "npm run lint && npm run test:code",
+    "test:code": "NODE_ENV=test mocha ./test",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix ."
+  },
+  "devDependencies": {
+    "mocha-it-each": "^1.0.2",
```
Comment (acamaragl) [inline package.json:17]: Mocha dependency is missing.
Maybe you have mocha installed globally.

Comment (yaaminu) [reply package.json:17]: good catch

**Review by vigan-abd — APPROVED** [review]

**Pull request merged**